Twig/TwigYuiExtension.php

```diff
@@ -90,6 +90,14 @@ public function yuiConfigLoaderFunction($configObject = '')
             'modules' => [],
         ];
 
+        $combine = $this->configResolver->getParameter('yui.combine', 'ez_platformui');
+
+        if (true === $combine) {
```

Comment:
I just looked it up, and learned something new today :) I don't like this either, but does the CS say anything about it? Whatever we choose, we should have consistency.